controllers/controllers/workloads/cfprocess_controller.go

```diff
@@ -103,6 +103,10 @@ func (r *CFProcessReconciler) Reconcile(ctx context.Context, req ctrl.Request) (
 			}
 		}
 
+		var lrpHealthCheckPort int32 = 0
```

Comment:
Isn't this redundant? The zero value for int32 is already 0 if I recall correctly.

Reply:
I guess I prefer to be explicit for now, we can change it later if people have concerns